app/controllers/api/alerts_statuses_controller.rb

```diff
@@ -0,0 +1,60 @@
+module Api
+  class AlertsStatusesController < BaseController
+    include Subcollections::AlertStatusStates
+
+    def update
+      if @req.action == "providers_alerts"
```

Comment:
this fields weird, a POST action "providers_alerts" to get info.  I'd just move this to metadata:

```
{
   ...,
   "data" : {
       "provider_alerts" : [
       ]
   }
}
```

with OPTIONS /api/providers